liqd_product/assets/scss/_variables.scss

```diff
@@ -30,11 +30,12 @@ $feedback-color-consideration: #ffae00;
 $spacer: 1em;
 $padding: 1rem;
 
-$font-family-sans-serif: "Arial", "Helvetica", sans-serif;
+$font-family-sans-serif: "Source Sans Pro", "Arial", "Helvetica", sans-serif;
```

Comment:
What is the purpose of Arial and Helvetica here? I get the feeling we are just dragging them along.